neps/nep-0330.md

```diff
@@ -1,12 +1,14 @@
 ---
 NEP: 330
 Title: Source Metadata
-Author: Ben Kurrek <ben.kurrek@near.org>
+Author: Ben Kurrek <ben.kurrek@near.org>, Osman Abdelnasir <osman@near.org>
 DiscussionsTo: https://github.com/near/NEPs/discussions/329
-Status: Review
+Status: Approved
 Type: Standards Track
 Category: Contract
+Version: 1.1.0
```

Comment:
@ori-near I have introduced NEPs versioning as part of the metadata. I will go over all of the approved NEPs and set the version to 1.0.0 if it is not set, and update NEP-001.